sql/mob_droplist.sql

```diff
@@ -26776,6 +26776,10 @@ INSERT INTO `mob_droplist` VALUES (3290,0,0,1000,16242,@VCOMMON); -- Dark Ixion:
 INSERT INTO `mob_droplist` VALUES (3290,0,0,1000,2654,@ALWAYS);   -- Dark Ixion: Horn
 INSERT INTO `mob_droplist` VALUES (3290,0,0,1000,2655,@ALWAYS);   -- Dark Ixion: Tail
 
+INSERT INTO `mob_droplist` VALUES (3291,0,0,1000,2813,@COMMON);   -- Atkorkamuy Aptant: Durus 
+INSERT INTO `mob_droplist` VALUES (3291,0,0,1000,2816,@COMMON);   -- Atkorkamuy Aptant: Tachus
+INSERT INTO `mob_droplist` VALUES (3291,0,0,1000,4101,@COMMON);   -- Atkorkamuy Water Crystal
```

Comment:
The water crystal shouldnt be placed here. its dependant on players having signet, and whatnot.

Reply:
Just for clarity here the crystal drops are determined by the `Element` column inside `mob_family_system`

Reply:
Ah perfect I was going to ask if and/or where I needed to put that then. Will take a look at mob_family_system, but it sounds like I can just remove it from the drop table and it should be fine if it is family dependent.